Source/Meadow.Contracts/Hardware/IMeadowDevice.cs

```diff
@@ -6,6 +6,15 @@
 namespace Meadow.Devices
 {
     public delegate void PowerTransitionHandler();
+    public delegate void TimeChangedEventHandler(DateTime utcTime);
+
+    public interface INtpClient
```

Comment:
I think NTP should be part of the MeadowOS bucket, not the device. It's an OS Service

Reply:
moved and checked in